schemas/iso19139/src/main/plugin/iso19139/index-fields/index-subtemplate.xsl

```diff
@@ -139,6 +139,12 @@
       </xsl:otherwise>
     </xsl:choose>
 
+    <xsl:variable name="ftitle" select="if (normalize-space(gmd:description/gco:CharacterString) != '')
+                          then gmd:description/gco:CharacterString
+                          else string-join(.//gco:Decimal, ', ')" />
+           store="true" index="true"/>
+    <xsl:copy-of select="gn-fn-index:add-field('ftitle', $ftitle)"/>
```

Comment:
Those fields are subtemplate specific, maybe we should prefix them otherwise in tools like Kibana they will be mixed with others and user will not really get what fname is or the difference between ftitle and resourceTitle ?